Marlin/src/gcode/queue.cpp

```diff
@@ -249,7 +249,9 @@ void ok_to_send() {
   if(Screen_send_ok[cmd_queue_index_r]) {
     ack_gcode_event(Screen_send_ok_opcode[cmd_queue_index_r], CommandLine[cmd_queue_index_r]);
     SNAP_DEBUG_SET_GCODE_LINE(CommandLine[cmd_queue_index_r]);
-    Screen_send_ok[cmd_queue_index_r] = false;
+    // Screen_send_ok[cmd_queue_index_r] = false;
```

Comment:
why comment this line？

Reply:
Screen_send_ok[cmd_queue_index_r] will set to false when the data in sending buffer have send out  actually.
Here keep Screen_send_ok[cmd_queue_index_r] as true means that there is data in buffer need to sendout.